examples_utils/benchmarks/command_utils.py

```diff
@@ -120,6 +120,36 @@ def get_benchmark_variants(benchmark_name: str, benchmark_dict: dict) -> list:
     return variants
 
 
+def remove_wandb_args(cmd: str) -> str:
+    """Remove all wandb args and their values from command strings.
+    """
+
+    new_arg_list = []
+    arg_list = shlex.split(cmd)
+    # Avoid out of bounds access attempt
+    arg_list.append("")
+    
+    skip = False
+    arg_pattern = re.compile("^[-]+")
+    for i in range(len(arg_list)):
+        if arg_pattern.match(arg_list[i]) and "wandb" in arg_list[i]:
+            # Also remove the values given to wandb args using a skip bool for
+            # next iteration
+            if not arg_pattern.match("^[-]+", arg_list[i+1]):
```

Comment:
this part not needed and should also have a check for i + 1 < len(arg_list)

Reply:
why is this part not needed? 

I'll change the range of the loop instead of checking for i + 1 since I've added a empty string to the end of the list

Reply:
ah didn't see the empty str, for the regex part, arg_pattern already has the pattern

Reply:
Ah yh right, i didnt see that!